code/game/objects/items/potions.dm

```diff
@@ -0,0 +1,191 @@
+/obj/item/potion
+	name = "potion"
+	desc = "This doesn't look like it does anything."
+	icon = 'icons/obj/potions.dmi'
+	icon_state = "red_minibottle"
+	w_class = W_CLASS_SMALL
+	slot_flags = SLOT_BELT
+	var/full = TRUE
+	var/fail_message = "<span class='notice'>Nothing happens, though your stomach is a little unsettled. It seems the potion isn't agreeing with you.</span>"
+
+/obj/item/potion/attack_self(mob/user)
+	if(full)
```

Comment:
Make this `if(!full) return` instead, and do it for all these cases.

Remove indentation when possible.